### ISSUE TITLE
Bobmed is incompatible with mining in its current state.

### ISSUE THREAD

**Issue by CoreFlare** [Skyrat-SS13/Skyrat13#2848]
## Round ID and Date:

ID 1784.

8th of July, Wednesday, 2020.

## Testmerges (If Applicable):

#2578

## Reproduction:

Some megafauna (the gladiator being the main example) have a stupidly high hit rate and have a nearly guaranteed chance to break/dislocate/disable a limb/your body in one hit. Note that before Bobmed, you could take the damage from the gladiator at later stages somewhat reliably, if you had enough healing to tank it. Now, if you get to last stage, your entire body gets disabled in a couple of hits, if not just dead.

All corpses from dead legions still process infections, spiking the time dilation to 50%+ on boxstation. This spams the log and runs the infection process (I assume) for **every single corpse on lavaland. EG: Over 50 by the end of the shift all with maximum damage and infection chance.**

## Possible Screenshots of the Issue in question:

[Legion corpses.](https://i.imgur.com/hhQ93hk.png)
[Broken megafauna.](https://i.imgur.com/qS14WWy.png)

**Comment by BobJoga** [comment]
I am going to fix these issues. Legion corpses no longer receive wounds, and i am working so megafauna don't cause wounds.

**Comment by BobJoga** [comment]
yeah nevermind mega-fun-a will cause wounds along with everything else in https://github.com/Skyrat-SS13/Skyrat13/pull/2945 no cap
